packages/code-editor/src/CodeEditor/CodeEditor.tsx

```diff
@@ -74,18 +130,82 @@ export const HvCodeEditor = ({
         },
       });
     });
-  }, [monaco, colorModes, selectedTheme, colors]);
+  }, [
+    monacoInstance,
+    colorModes,
+    selectedTheme,
+    colors?.type,
+    colors?.atmo1,
+    colors?.secondary_60,
+  ]);
 
   useEffect(() => {
-    defineActiveThemes();
-  }, [selectedTheme, colorModes, defineActiveThemes]);
+    handleActiveThemes();
+  }, [selectedTheme, colorModes, handleActiveThemes]);
```

Comment:
The blame on these useless dependencies is on me - let's not pass it onto you 😅 
```suggestion
  }, [handleActiveThemes]);
```

Reply:
I'll remove them 🙏